src/main/java/ch/njol/skript/classes/data/DefaultComparators.java

```diff
@@ -253,10 +244,10 @@ public boolean supportsOrdering() {
 		// Block - Block
 		Comparators.registerComparator(Block.class, Block.class, new Comparator<Block, Block>() {
 			@Override
-			public Relation compare(final Block b1, final Block b2) {
-				return Relation.get(b1.equals(b2));
+			public Relation compare(Block b1, Block b2) {
+				return Relation.get(BlockUtils.getBlock(b1).equals(BlockUtils.getBlock(b2)));
```

Comment:
Instead of creating a utility method, should DelayedChangeBlock instead override equals?

Reply:
That would only work one way around, `DelayedChangeBlock#equals(CraftBlock)`, but not for `CraftBlock#equals(DelayedChangeBlock)`.

I am going to rename the utility method to extractBlock, getBlock isn't descriptive enough